security/fma-superchainweth.md

```diff
@@ -0,0 +1,77 @@
+- **Detection:** Ideally, off-chain scripts should detect if multiple `mint` calls revert. Additionally, check the current `ETHLiquidity` balance. If that’s not feasible, user-filed support tickets may flag this issue, since funds are not at risk of loss due to this failure.
+- **Recovery Path(s):** Coordinate an L2 hard fork to replenish the `ETHLiquidity` contract. Investigate the causes of the depletion to determine if other factors are involved.
+
+### FM2: Overflow during `burn` in `ETHLiquidity` due to max Balance
+
+- **Description:** If the `ETHLiquidity` contract has already reached the maximum allowed ETH balance (`type(uint256).max`), invoking the `sendETH` function (which triggers `crosschainBurn`) could cause an overflow. This occurs when `burn` is called with an amount that, when added to the current balance, exceeds the maximum `uint256` value. Such an overflow would cause a revert, since the proper Solidity version is used.
+- **Risk Assessment:** Low.
+    - Potential Impact: Medium. Reverts during `sendETH` calls are expected when the requested amount exceeds the maximum value representable by a `uint256`.
+    - Likelihood: Very low. `ETHLiquidity` starts with a maximum balance (`type(uint248).max`) which is still far from `uint256` limit.
+- **Mitigations:** Initial ETH supply in `ETHLiquidity` is `type(uint248).max` properly is checked ([test](https://github.com/ethereum-optimism/optimism/blob/dd37e6192c37ed4c5b18df0269f065f378c495cc/packages/contracts-bedrock/test/L2/ETHLiquidity.t.sol#L29)).
+- **Detection:** Given the unlikely nature and potential impact of this case, user-filed support tickets should be sufficient to flag this issue since funds are not at risk of loss due to this failure.
+- **Recovery Path(s):** Coordinate an L2 hard fork to adjust the `ETHLiquidity` contract’s state. Investigate the causes of this failure to determine if other factors are involved.
+
+### Generic items we need to take into account:
+
+See [fma-generic-contracts.md](https://github.com/ethereum-optimism/design-docs/blob/main/security/fma-generic-contracts.md).
+
+- [x]  Check this box to confirm that these items have been considered and updated if necessary.
+
+See [relevant FMAs to SuperchainWETH, To Do] 
+
+- [ ]  Check this box to confirm that these items have been considered and updated if necessary.
+
+## Action Items
+
+- [ ]  Resolve all the comments.
+- [ ]  FM1: Decide whether to use off-chain scripts or rely on a user-support system for detection.
+- [ ]  Ensure the support team is aware of these failure modes and prepared to respond.
+- [ ]  **Ensure that the actions items specified in each FMAs on which SuperchainWETH depends are completed.**
```

Comment:
Optional action item to have monitoring that detects Mint events without Burn events

Reply:
^ Worth adding into action items but mark as optional